src/cli/bin.ts

```diff
@@ -100,21 +105,38 @@ yargs
       debug,
       stack,
     }) => {
+      const emptyUIFolder = emptyUIFolder50;
+      const migratedUIFolder: ContentRecord = _cloneDeep(emptyUIFolder);
```

Comment:
We shouldn't have this.
Each migration function should just receive `contentServer`.